ante/ante.go

```diff
@@ -46,14 +46,20 @@ func NewAnteHandler(opts HandlerOptions) (sdk.AnteHandler, error) {
 		sigGasConsumer = ante.DefaultSigVerificationGasConsumer
 	}
 
+	// maxBypassMinFeeMsgGasUsage is the maximum gas usage per message
+	// so that a transaction that contains only message types that can
+	// bypass the minimum fee can be accepted with a zero fee.
+	// For details, see gaiafeeante.NewFeeDecorator()
+	var maxBypassMinFeeMsgGasUsage uint64 = 200_000
```

Comment:
Nit: consider making this a const and defining it at the top of the file for clarity.

Reply:
We want to refactor the entire thing anyway, so I don't think it's important